interface/forms/eye_mag/a_issue.php

```diff
@@ -705,14 +643,14 @@ function negate_radio(section) {
                         $title = "Eye Medications";
                     }
 
-                    $HELLO[attr($key) ] = '<input type="radio" name="form_type" id="PMSFH_' . attr($key) . '" value="' . attr($key) . '" ' . $checked . ' onclick="top.restoreSession();newtype(\'' . attr($key) . '\');" /><span>' . '<label class="input-helper input-helper--checkbox" for="PMSFH_' . attr($key) . '" title="' . xla($title) . '">' . xlt($key_short_title) . '</label></span>&nbsp;';
+                    $HELLO[attr($key) ] = '<input type="radio" name="form_type" id="PMSFH_' . attr($key) . '" value="' . attr($key) . '" ' . $checked . ' onclick="top.restoreSession();newtype(\'' . attr($key) . '\');" /><span>' . '<label class="input-helper input-helper--checkbox" for="PMSFH_' . attr($key) . '" title="' . xla($title) . '" />' . xlt($key_short_title) . '</label></span>&nbsp;';
                 }
 
 //put them in the desired display order
                 echo $HELLO['POH'] . $HELLO['POS'] . $HELLO['Eye Meds'] . $HELLO['PMH'] . $HELLO['Medication'] . $HELLO['Surgery'] . $HELLO['Allergy'] . $HELLO['FH'] . $HELLO['SOCH'] . $HELLO['ROS'];
                 ?>
             </div>
-            <div class="borderShadow" style="text-align:left;">
+            <div class="borderShadow text-left">
                 <table class='border-0 w-100'>
                     <tr id='row_quick_picks'>
                         <td valign='top' nowrap>&nbsp;</td>
```

Comment:
Replace nowrap with `text-nowrap` class and replace `valign='top'` with `align-top` BS4 class

This file needs a lot more work